### DISCUSSION TITLE
启动be报错

### DISCUSSION THREAD

**Discussion by yimingtech** [apache/doris#33631] [RFC]
在docker 环境中，运行./bin/start_be.sh --daemon 会报Please disable swap memory before installation, eg: 'swapoff -a'. 执行 swapoff -a的时候会提示 swapoff: Not superuser. 这个咋搞？

**Comment by Vallishp** [comment]
you need to turnoff swap before creating the docker container instance.
1)exit out of the docker container
2) delete your current docker container (docker rm name)
3) sudo swapoff  -a
4) create docker container instance 
5) attach with docker and start fe and be.

if you don't want to delete the docker container for some reason, you can modify start_be.sh and remove exit 1 after the swap check.

    if [[ "$(swapon -s | wc -l)" -gt 1 ]]; then
        echo "Please disable swap memory before installation."
        exit 1 // remove it. it will impact  performance. but normal verification still can be done.
    fi
